ui/js/signup.js

```diff
@@ -199,6 +202,13 @@ require(['jquery', 'underscore', 'oae.core'], function($, _, oae) {
             return;
         }
 
+        // Get the email domain for the tenant (if any) so we can validate the email
+        oae.api.admin.getTenant(null, function(err, tenant) {
+            if (!err && tenant.emailDomain) {
+                emailDomain = new RegExp(tenant.emailDomain + '$');
```

Comment:
Wouldn't this cause `bla@fooox.ac.uk` to be a valid address when `ox.ac.uk` is configured as the email domain?